README.md

```diff
@@ -86,7 +86,7 @@ Example output from an ss3sim simulation
 
 ![An example of ss3sim output](https://raw.github.com/seananderson/ss3sim/master/inst/ms/fig2-20131109.png)
```

Comment:
Should this link be within ss3sim/ss3sim instead? Just curious... (I know it's not part of the PR, really)

Reply:
Moved to separate issue. Thanks for noticing.